index.html

```diff
@@ -11,17 +11,24 @@
 <body>
 
     <header>
-       
+        <nav>
+            <ul>
+              <li><a href="#landing">Home</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#services">Services</a></li>
+              <li><a href="#contact">contact</a></li>
+            </ul>
+        </nav>
     </header>
 
-    <nav>
-        <ul>
-          <li><a href="#landing">Home</a></li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#services">Services</a></li>
-          <li><a href="#contact">contact</a></li>
-        </ul>
     <section id="landing">
+        <div class="grid-container center width-lg">
```

Comment:
html structure is neat and semantic 😃  smoothly justifying between each div tag and its attributes, nicely written